erreser.go

```
@@ -0,0 +1,624 @@
+		return
+	}
+	mf := models.Feed{
+		Title:       feedData.Title,
+		Description: feedData.Description,
+		Link:        feedData.Link,
+		Feed:        feed,
+	}
+
+	dbParams := database.InitDbParams()
+	db := database.DbConnect(dbParams)
+	db.AutoMigrate(&models.Feed{})
```

golangci-lint (1.62.2)

Error return value of `db.AutoMigrate` is not checked

(errcheck)